### ISSUE TITLE
Smooth corners for natural polygons

### ISSUE THREAD

**Issue by Explorer256** [organicmaps/organicmaps#2914]
I suggest smoothing the corners of natural polygons (grass, forests, ponds, etc.). 

This will make the map look cleaner and more modern.

**Comment by bveldkamp** [comment]
I don't think this is a good idea. For polygons this may give strange results with neighboring parcels. E.g. what should the background color of the rounded part in this situation be:

![afbeelding](https://user-images.githubusercontent.com/7160755/181781466-21afc9ca-e37f-4f3e-8078-5b4ccb805e20.png)

It *might* be an idea for rivers, but still, I think if there actually *is* a straight corner in a polyline, I would like to see the exact shape on the map rather than an arbitrary rounded shape, just because it would look nice.
